google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITTransferManagerTest.java

```diff
@@ -79,5 +109,14 @@ public void uploadFiles() throws IOException {
   }
 
   @Test
-  public void downloadBlobs() {}
+  public void downloadBlobs() throws IOException {
+    TransferManagerConfig config =
+        TransferManagerConfig.newBuilder().setAllowChunking(false).setMaxWorkers(1).build();
```

Comment:
Isn't setAllowChunking false by default?

Reply:
yes it is, can remove that.